src/main/java/org/myrobotlab/service/WebGui.java

```diff
@@ -637,7 +639,11 @@ public void handle(AtmosphereResource r) {
 
       } else if (apiKey.equals(CodecUtils.API_SERVICE)) {
 
-        Message msg = CodecUtils.cliToMsg(null, getName(), null, r.getRequest().getPathInfo());
+        Message msg = CodecUtils.cliToMsg(
+                null,
+                getName(),
+                null,
+                URLDecoder.decode(r.getRequest().getPathInfo(), StandardCharsets.UTF_8));

```

Comment:
Great ! ...
what's the use case that broke this, that you fixed ?
(should be added to unit test)

Reply:
This section adds the ability to use URI path encoding schemes in the service API. So double quotes are %22 and will be transparently decoded to double quotes by this section. I'll add it to the test suite